satellite/metainfo/metainfo.go

```diff
@@ -43,26 +44,28 @@ type APIKeys interface {
 
 // Endpoint metainfo endpoint
 type Endpoint struct {
-	log           *zap.Logger
-	metainfo      *Service
-	orders        *orders.Service
-	cache         *overlay.Cache
-	apiKeys       APIKeys
-	accountingDB  accounting.DB
-	maxAlphaUsage memory.Size
+	log               *zap.Logger
+	metainfo          *Service
+	orders            *orders.Service
+	cache             *overlay.Cache
+	apiKeys           APIKeys
+	accountingDB      accounting.DB
+	accountingRTCache accountingcache.Service
+	maxAlphaUsage     memory.Size
```

Comment:
Should we remove this `Alpha` here to be `maxUsage`?

Reply:
Good call!